src/scenes/home/landing/hero/hero.css

```diff
@@ -0,0 +1,77 @@
+.heroContainer {
+  display: flex;
+  flex-flow: column;
+}
+
+.pageHeading {
+  background-color: #191f39;
+  color: #ffffff;
+  text-align: center;
+  padding: 15px;
```

Comment:
The `.pageHeading` div appears very close to the video on its right. The horizontal padding should probably be increased, something like to `padding: 15px 30px`, to give it some breathing room.